buildtest/cli/__init__.py

```diff
@@ -399,6 +408,28 @@ def misc_menu(subparsers):
         help="Show help message for command",
     )
 
+    profile_argparse = subparsers.add_parser(
+        "help",
+        config=["-h"],
+        compliers="config profiles list",
+    )
+    profile.add_list(
+        "command",
+        choices=[
+            "compilers",
+            "co",
+            "edit",
+            "e",
+            "executors",
+            "ex",
+            "path",
+            "p",
+            "systems",
+            "validate",
+            "view",
+            "-v"
+        ],
+    )
```

Comment:
This could potentially be correct, I'll take a closer look.

Reply:
already done